WebService/WebService/Base Classes/SpecificPrice.cs

```diff
@@ -7,8 +7,11 @@ namespace WebService.Base_Classes
 {
     public class SpecificPrice
     {
+        public int SpecificPriceID { get; set; }
```

Comment:
I don't know if having a PK for SpecificPrice won't give us trouble, because the way to get SpecificPrice will be mainly through PK and it won't say if BarID and DrinkID matches, so we will need to do it programatically